demos/ci-cd-ops-role-sdp/README.md

```diff
@@ -0,0 +1,30 @@
+# Enhance CICD Demo to Add the "OPs" side of the story - Trello card # 132
```

Comment:
I would also include in this readme a list of Pre-Requisutes (i.e. an OpenShift environment with...) & assumptions about the potential presenter (what you assume he/she knows & is capable of).